### PR TITLE
Update evmtraining2 20230511

### PR DESCRIPTION
**Pull request by lee920217**
- change the document structure 
- add the missing file 
- update some error path

### CODE REVIEW

File: docs/evm_training/evmTask2.md [hunk 1]
```diff
@@ -8,7 +8,14 @@ Compiling and deploying a smart contract on Nervos' EVM compatible [Layer 2](str
 
 ## Task Instructions
 
-In this task, you will take a very basic smart contract written in Ethereum's Solidity, compile it to EVM bytecode and deploy it to Godwoken on Layer 2.
+In this task, you will 
+<div class="task-wrapper">
+
+- Take a very basic smart contract written in Ethereum's Solidity.
```
Comment (Flouse) [inline docs/evm_training/evmTask2.md:14]:
```suggestion
- Take a very basic smart contract written in [Solidity](https://soliditylang.org/).
```

File: docs/evm_training/evmTask2.md [hunk 2]
```diff
@@ -8,7 +8,14 @@ Compiling and deploying a smart contract on Nervos' EVM compatible [Layer 2](str
...
+- Compile it to EVM bytecode.
+- Deploy it to Godwoken on Layer 2.
```
Comment (Flouse) [inline docs/evm_training/evmTask2.md:16]:
```suggestion
- Deploy it to [Godwoken Testnet](https://v1.testnet.gwscan.com/)
```

File: docs/evm_training/evmTask2.md [hunk 3]
```diff
@@ -86,6 +97,36 @@ contract SimpleStorage {
 }
 ```
 
+### 3. Deploy a Smart Contract
+
+Hardhat will handle the process of compilation and deployment of the smart contract.
```
Comment (Flouse) [inline docs/evm_training/evmTask2.md:102]:
```suggestion
[Hardhat](https://hardhat.org/) will handle the process of compilation and deployment of the smart contract.
```

File: docs/evm_training/evmTask2.md [hunk 4]
```diff
@@ -17,10 +24,13 @@ The instructions will provide a basic smart contract you can work with, but you
 ### Prerequisites
 
 Before you begin this task, you must complete the [previous task](evm_training/evmTask1.md) to create a Godwoken account on the EVM Layer 2 Testnet. You will also need the private key and Testnet address from the previous task in order to proceed. If you have not completed it, please do so now.
+<div class="task-wrapper">
 
-### 1. Prepare Your Ethereum Private Key
+- [Godwoken Account on EVM layer 2 Test](evm_training/evmTask1.md)
+- [Private Key](https://docs.godwoken.io/evm_training/evmTask1#1-go-to-godwoken-bridge)
+- [Testnet address](/evm_training/evmTask1#1-go-to-godwoken-bridge)
```
Comment (Flouse) [inline docs/evm_training/evmTask2.md:31]: Testnet address is not used in this page?

File: docs/evm_training/evmTask2.md (hunk 2 above)
Comment (lee920217) [reply docs/evm_training/evmTask2.md:16]: done

File: docs/evm_training/evmTask2.md (hunk 1 above)
Comment (lee920217) [reply docs/evm_training/evmTask2.md:14]: done

**Review by Flouse — APPROVED** [review]

**Pull request merged**